settings.gradle

```diff
@@ -1,3 +1,3 @@
 rootProject.name = "jsonassert-root"
 
-include 'jsonassert-assertj-java7', 'jsonassert-assertj-java8', 'jsonassert'
\ No newline at end of file
+include 'jsonassert-assertj-java7', 'jsonassert-assertj-java8', 'jsonassert', 'jsonassert-shadow'
```

Comment:
Any reason to not just do this shading in `jsonassert` in major version bump so that from that point on, everyone gets shaded version?

Reply:
Maybe you don't want to have the shaded version really. That way we don't break anyone - we just give one more option of using this tool

Reply:
I figured that was the motivation. That makes sense, but as you mentioned [here](https://github.com/marcingrzejszczak/jsonassert/pull/36#discussion_r531733836) that may not be a big concern. If that is true then personally I would lean on the less is more here - it would be one less sub-module / build to maintain. But really, either way, not a big deal.